src/main.js

```diff
@@ -115,4 +115,6 @@ function createPoster(imageURL, title, quote) {
     imageURL: imageURL, 
     title: title, 
     quote: quote}
-}
\ No newline at end of file
+}
+
+// Hey this a test comment for our test branch! //
```

Comment:
Here is that test comment in the main file!

Reply:
Looks good!